cardano-cli/test/cardano-cli-golden/files/golden/allegra/transaction-view.out

```diff
@@ -22,5 +22,6 @@ return collateral: null
 total collateral: null
 update proposal: null
 validity range:
-  time to live: 101
+  lower bound: null
+  upper bound: 101
```

Comment:
As a result, we also show upper bound and never show time to live in `transaction view`.